docs/custom_tests.rst

```diff
@@ -215,3 +215,44 @@ module directories:
 .. code-block:: console
 
     $ memote run -a "--durations=10 path/to/dir1/ path/to/dir2/" --filename "report.html" path/to/model.xml
+
+
+Guidelines
+==========
+
+Please consider the following guidelines which reflect some of the considerations
+behind the core tests in memote. Adhering to these guidelines will allow other
+researchers to easily adopt your custom tests and ensure that they are applicable
+to a wide array of modeling practises.
+
+1. **Be namespace agnostic**. Use the ``METANETX_SHORTLIST`` and
```

Comment:
Consider including a link to where exactly this can be found (as well as `COMPARTMENT_SHORTLIST` below) so the reader doesn't have to search.

Reply:
Good call! Added.